middle_end/flambda2/terms/effects_and_coeffects.mli

```diff
@@ -24,15 +24,19 @@ val print : Format.formatter -> t -> unit
 val compare : t -> t -> int
 
 (** The value stating that no effects of coeffects take place. This is exactly
-    [No_effects, No_coeffects]. *)
+    [No_effects, No_coeffects, Strict]. *)
 val pure : t
 
+(** The value stating that no effects of coeffects take place. This is exactly
+    [No_effects, No_coeffects, Delay]. *)
+val pure_duplicatable : t
```

Comment:
Can I suggest `pure_can_be_duplicated`?  `duplicatable` is a weird word and a bit of a mouthful.

Reply:
done